app/assets/stylesheets/components/page_specific/_promo.scss

```diff
@@ -9,19 +9,72 @@
 .promo__heading {
   @extend %font-heading-medium;
   @include body(18, 24);
-  margin-top: 0;
+  color: $color-blue-medium;
+  margin: 0 0 $baseline-unit;
+  position: relative;
 
   @include respond-to($mq-m) {
     @include body(20, 24);
   }
+
+  .l-tool-promos__item & {
+    padding-right: $baseline-unit*3;
+  }
+}
+
+.promo__heading--blog:after {
+  background: $color-mas-blog-blue;
+  border-radius: 4px;
+  bottom: 0;
+  color: $color-white;
+  content: "Blog";
+  display: inline-block;
+  font-size: 18px;
+  font-weight: 700;
+  height: 30px;
+  left: 37%;
+  padding: $baseline-unit/2 $baseline-unit*2.5;
+  position: absolute;
+  text-transform: uppercase;
+  top: -$baseline-unit*6;
+}
+
+.promo__link {
+  &:hover,
+  &:focus {
+    text-decoration: none;
+
+    .promo__heading {
+      text-decoration: underline;
+    }
+  }
+
+  .l-promo-articles__item--no-image & {
```

Comment:
Selector `l-promo-articles__item--no-image` should be written in lowercase with hyphens